cartographer/mapping/internal/3d/pose_graph_3d_test.cc

```diff
@@ -109,14 +108,16 @@ TEST_F(PoseGraph3DTest, BasicSerialization) {
   proto::PoseGraph proto;
   auto fake_node = test::CreateFakeNode();
   test::AddToProtoGraph(fake_node, &proto);
-  pose_graph_->AddNodeFromProto(transform::Rigid3d::Identity(), fake_node);
+  pose_graph_->AddNodeFromProto(Rigid3d::Identity(), fake_node);
   auto fake_submap = test::CreateFakeSubmap3D();
   test::AddToProtoGraph(fake_submap, &proto);
-  pose_graph_->AddSubmapFromProto(transform::Rigid3d::Identity(), fake_submap);
+  pose_graph_->AddSubmapFromProto(Rigid3d::Identity(), fake_submap);
   test::AddToProtoGraph(test::CreateFakeConstraint(fake_node, fake_submap),
                         &proto);
   pose_graph_->AddSerializedConstraints(FromProto(proto.constraint()));
-
+  test::AddToProtoGraph(
```

Comment:
Maybe add a EXPECT_TRUE(MessageDifference for `.landmarks()` so it is easier to debug in case the test fails.